### DISCUSSION TITLE
post/14-unraid-proxy-using-dns/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#5] [Announcements]
# post/14-unraid-proxy-using-dns/

前言最近升级了 HomeLab 的 Unraid 系统到最新的 6.11.5 版本(之前的破解方法也失效了, 新方法自行搜索), 许多插件和 Docker 容器在运行了半年后已经需要升级. 可是最近 GitHub 的访问质量越来越差, 升级时的网络问题还是需要解决. 然而 Unraid 系统设置系统代理相对复杂, 所以将我目前的解决方案记录在此. Docker 容器更新方法Docker 其实只需要

https://blog.lyc8503.site/post/14-unraid-proxy-using-dns/

**Comment by lyc8503** [comment]
后来发现文中提到的代理方法其实已经有人实现过了(⁠╯⁠°⁠□⁠°⁠）⁠╯⁠︵⁠ ⁠┻⁠━⁠┻  叫 sniproxy.
